html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <div class="coding-form-container">
+    <form action="index.html" method="get">
+      <div class="email-section">
+        <label>Email<br>
+          <input type="email" name="email">
+        </label>
+      </div>
+      <div class="why-cool-container">
+        <label for="why-cool">Why is coding cool?<br>
+          <select name="coding-is-cool" id="why-cool">
+            <option value="select-one" selected="selected">Select an option</option>
+            <option value="interesting">Because it's interesting</option>
+            <option value="fun">Because it's fun</option>
+            <option value="powerful">Because it's powerful</option>
+          </select>
+        </label>
+      </div>
+      <div class="fav-lang-container">
+        <p>
+          Favorite Language (choose one)
+          <br>
+          <label>
+            <input type="radio" name="lang" value="HTML">HTML
+          </label>
+          <label>
+            <input type="radio" name="lang" value="CSS"> CSS
+          </label>
+          <label>
+            <input type="radio" name="lang" value="JavaScript"> JavaScript
+          </label>
+        </p>
+      </div>
+      <div class="sign-up-section">
+        <label for="sign-up"> Would you like to sign up for more information?</label>
```

Comment:
This is actually just general text describing the section below since it isn't labeling one specific form control element. You should use a `p` element.